code/modules/admin/topic.dm

```diff
@@ -114,39 +114,39 @@
 		switch(bantype)
 			if(BANTYPE_PERMA)
 				if(!banckey || !banreason)
-					to_chat(usr, "Not enough parameters (Requires ckey and reason)")
+					to_chat("<span class='warning'>Not enough parameters (Requires ckey and reason)</span>")
```

Comment:
missing all the `usr` vars here. Best to go over the commit again and re add the usr params